### PR TITLE
Complete Send_Scheduler_Ack task

### PR DESCRIPTION
**Pull request by imsickofmaps**
Currently placeholder - should fire DELETE request to message-id

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr ready for review

File: mama_ng_control/apps/vumimessages/tests.py [hunk 1]
```diff
@@ -319,8 +323,16 @@ def test_delete_inbound_data(self):
...
+    @responses.activate
     def test_event_ack(self):
         existing = self.make_outbound()
+
+        # Setup response from scheduler
+        responses.add(
+            responses.DELETE,
+            "http://127.0.0.1:8000/mama-ng-scheduler/rest/messages/1",
+            json.dumps({}), status=200, content_type='application/json')
+
```
Comment (gsvr) [inline mama_ng_control/apps/vumimessages/tests.py:335]: Do you want to add an assert here somewhere to check that the subscription metadata for scheduler_message_id has been set to ""?

Comment (imsickofmaps) [inline mama_ng_control/apps/vumimessages/tests.py:335]: Done

**Comment by gsvr** [comment]
:+1:

**Pull request merged**